packages/website/src/pages/mentor.tsx

```diff
@@ -1,11 +1,13 @@
 import { MentorPerspective } from '@/components/MentorPerspective';
 import { MentorRequirements } from '@/components/MentorRequirements';
+import { MentorSchedule } from '@/components/Schedule/MentorSchedule/MentorSchedule';
 
 const Mentor = () => {
   return (
     <>
-      <MentorPerspective />
       <MentorRequirements />
+      <MentorPerspective />
```

Comment:
Why MentorPerspective is moved under MentorRequirements ?

Reply:
Ups sorry my bad